cdap-ui/cypress/fixtures/fll_wrangler-test-pipeline.json

```diff
@@ -1,28 +1,28 @@
 {
-    "name": "Airport_test",
-    "description": "Data Pipeline Application",
     "artifact": {
         "name": "cdap-data-pipeline",
-        "version": "[6.1.0-SNAPSHOT, 6.2.0-SNAPSHOT]",
+        "version": "6.2.0-SNAPSHOT",
```

Comment:
provide version ranges for all versions in this file. Otherwise we will have to update the version every time